client/src/components/UserDashBoard.jsx

```diff
@@ -1,16 +1,121 @@
 const React = require('react');
 const UserDashBoardKeywordsEntryBar = require('./UserDashBoardKeywordsEntryBar.jsx');
+const UserDashBoardMonitoredWordResult = require('./UserDashBoardMonitoredWordResult.jsx');
 
 class UserDashBoard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isFetching: true,
+      monitoringResults: this.props.monitoringResults,
+      errorMessage: ''
+    };
+
+    this.handleAddMonitoredWords = this.handleAddMonitoredWords.bind(this);
+    this.handleRemoveMonitoredWords = this.handleRemoveMonitoredWords.bind(this);
+  }
+  componentDidMount() { 
+    // Retrieve user monitored keywords and associated bills to populate the dashboard
+    $.get('user/' + this.props.username + '/keywords')
+      .done(data => this.setState({
+        isFetching: false,
+        monitoringResults: data,
```

Comment:
data will be an object with the following format:
{keywords: {"keyword1": {keyword: "keyword1",
                                        associatedKeywords: [...,...,],
                                        bills: [id1, id2, id3]}
                  }